scripts/monitor/visualization/main.py

```diff
@@ -0,0 +1,43 @@
+#!
+import pickle
+
+from template import *
+from week_one import *
+from week_two import *
+
+ds_meta_runtime = pickle.load(open("monitoring_dataset_50e68afe-a342-4357-9397-4de7b9300b10.metadata_runtime.pickle", "rb"))
+ds_metrics = pickle.load(open("monitoring_dataset_50e68afe-a342-4357-9397-4de7b9300b10.metrics.pickle", "rb"))
+# above is very temporary measure
+
+
+def generate_individual_plots():
+    pass
+
+
+def generate_report_homepage(meta_runtime: pd.DataFrame, metrics: pd.DataFrame):
+
+    html_template = HTML_STYLE+HTML_HEADING+generate_html_tables_section(table_html(meta_runtime, metrics))+HTML_VIOLIN+HTML_SCATTER
+    with open('home.html', 'w') as home:
+        #home.write('') #alternate
+        home.write(html_template)
+
+
+def table_html(meta_runtime: pd.DataFrame, metrics: pd.DataFrame):
+    dicts = num_task_shards(meta_runtime)
```

Comment:
let's not use generic names as `dict`'s, but rather a name indicating what's in it.